File: apps/marquee/quorum-sense/ios/Capture/CaptureSessionContext.swift
import Foundation

/// Session-scoped capture context injected from the host app (M3.2).
public enum CaptureSessionContext {
    private static let defaultInquiryThreadId = "inq_mobile_launch_risks"

    /// Resolve the active inquiry thread from scheme env or fall back to the demo id.
    public static func inquiryThreadId() -> String {
        ProcessInfo.processInfo.environment["QUORUM_INQUIRY_THREAD_ID"] ?? defaultInquiryThreadId
    }
}

File: apps/marquee/quorum-sense/android/app/src/main/java/se/reflective/quorum/capture/CaptureSessionContext.kt
package se.reflective.quorum.capture

/** Session-scoped capture context injected from the host app (M5.7 / M3.2 parity). */
object CaptureSessionContext {
    private const val DEFAULT_INQUIRY_THREAD_ID = "inq_mobile_launch_risks"

    fun inquiryThreadId(): String =
        System.getenv("QUORUM_INQUIRY_THREAD_ID") ?: DEFAULT_INQUIRY_THREAD_ID
}
